config/en.yml

```diff
@@ -6,89 +6,89 @@ en:
 
 ########################################
 ### en/account.yml
-  congratulations_you_have_signed_up: Congratulations! You have signed up for %{site_title}.
-  cookie_disabled_warning: You have cookies disabled. You will not be able to login
-    until you enable cookies.
-  email_verification_body: |-
+  congratulations_you_have_signed_up: "Congratulations! You have signed up for %{site_title}.
```

Comment:
The quotation marks `"` include the following line in this translation. But the `cookie_disabled_warning` is a separate string all together.